images/users/zimp.svg

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
As explained at the bottom of the user page (http://babeljs.io/users/), you need to provide an otpimized SVG.